### PR TITLE
OSDOCS-2279: DNS Operator managementState release note

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-2279

Related to https://github.com/openshift/openshift-docs/pull/36197 and https://github.com/openshift/openshift-docs/issues/33497. 

Preview: 
https://deploy-preview-36871--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-9-networking-dns-management-state

### CODE REVIEW

**Comment by bmcelvee** [comment]
This is the release note for DNS Operator `managementState`. 

@lihongan and @rfredette would you mind giving this PR a quick review, please? Thank you!

**Comment by lihongan** [comment]
LGTM.
Thank you @bmcelvee

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -334,6 +334,19 @@ This release introduces six timeout configurations for the Ingress Controller `t
...
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress controller configuration parameters].
 
+[id="ocp-4-9-networking-dns-management-state"]
+==== DNS Operator managementState
+
+In {product-title} {product-version}, you can now change the DNS Operator `managementState`. The `managementState` of the DNS Operator is set to `Managed` by default, which means that the DNS Operator is actively managing its resources. You can change it to `Unmanaged`, which means the DNS Operator is not managing its resources.
+
+The following are use cases for changing the DNS Operator `managementState`:
+
+* You are a developer and want to test a configuration change to see if it fixes an issue in CoreDNS. You can stop the DNS Operator from overwriting the fix by setting the `managementState` to `Unmanaged`.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-9-release-notes.adoc:344]:
```suggestion
* You are a developer and want to test a configuration change to see if it fixes an issue in CoreDNS. You can stop the DNS Operator from overwriting the change by setting the `managementState` to `Unmanaged`.
```

**Review by jeana-redhat — APPROVED** [review]
Just one tiny suggestion, LGTM

**Pull request merged**